README.md

```diff
@@ -1,57 +1 @@
-
-The Tellor system is a way to push data on-chain. What pieces of data are pushed are specificied in the `configs/indexes.json`. Note that the data corresponds to a specific API. The tellor mining system is set up to pull api data to generate these values to submit on-chain once a correct nonce is mined. These specific apis are just suggestions. The system is not guarunteed to work for everyone. It is up to the consnesus of the Tellor token holders to determine what a correct value is. As an example, request ID 4 is BTC/USD. If the api's all go down, it is the responsibility of the miner to still submit a valid BTC/USD price. If they do not, they risk being disputed and slashed. For these reasons, please contribute openly to the official Tellor miner (or an open source variant), as consensus here is key. If you're miner gets a different value than the majority of the of the other miners, you risk being punished.
-
-A list of all PSR's(pre specified requests) and the expected data can be found [here](blob/master/pkg/tracker/psrs.go).
-
-<p align="center">
-    <img src= './assets/minerspecs.png' width="450" alt='MinerSpecs' />
-</p>
-
-
-## Instructions for deployment
- - [Telliot Instructions](https://app.gitbook.com/@tellor-2/s/tellor-docs/miner-documentation)
-
-## Contributing and building from source
-Contributions are very welcome!<br/>
-See our [contributing.md](docs/contributing.md) for more information.
-
-## Run with Docker
-Available tags: https://hub.docker.com/u/tellor
-docker run -v $(pwd)/.local:/configs tellor/telliot:master mine
-
-## Maintainers <a name="maintainers"> </a>
-This repository is maintained by the [Tellor team](https://github.com/orgs/tellor-io/people)
-
-### DISCLAIMER
-    Mine at your own risk.
-
-    Mining requires you deposit 1000 Tellor Tributes. These are a security deposity. If you are a malicious actor (aka submit a bad value), the community can vote to slash your 1000 tokens.
-
-    Mining also requires submitting on-chain transactions on Ethereum. These transactions cost gas (ETH) and can sometimes be signifiant if the cost of gas on EThereum is high (i.e. the network is clogged). Please reach out to the community to find the best tips for keeping gas costs under control or at least being aware of the costs.
-
-    If you are building a competing client, please contact us. The miner specifications are off-chain and the validity of the mining process hinges on the consensus of the Tellor community to determine what proper values are. Competing clients that change different pieces run the risk of being disputed by the commmunity.
-
-    There is no guaruntee of profit from mining.
-
-    There is no promise that Tellor Tributes currently hold or will ever hold any value.
-
-
-### Copyright
-Tellor Inc. 2019
+See [docs](https://docs.tellor.io/miner-documentation/miner)
```

Comment:
I am thinking to add 2 links here like
```suggestion
The project uses Gitbook linked to the [docs](./docs) folder of the repo.
[Telliot Docs](https://docs.telliot.tellor.io)
[Tellor Project](https://docs.tellor.io)

```

Reply:
I added this just an example, can you please add a bit better formating.

https://github.com/hhio618/telliot/tree/move-md-to-docs

Also I just noticed that the links to the assets also needs updating
https://github.com/hhio618/telliot/tree/move-md-to-docs/docs
The images are broken but probably best to fix these with the refactoring of the Readme page anyway.

Reply:
Ok sure.
I'll fix all broken links in a separate PR.